examples/veracruz_demo/example.toml

```diff
@@ -0,0 +1,16 @@
+name = "veracruz_demo"
+
+[applications]
+
+[applications.rust]
+manifest = "examples/veracruz_demo/oak_app_manifest.toml"
+out = "examples/veracruz_demo/bin/veracruz_demo.oak"
+
+[applications.rust.modules]
+module = { Cargo = { cargo_manifest = "examples/veracruz_demo/module/rust/Cargo.toml" } }
+translator = { Cargo = { cargo_manifest = "examples/translator/module/rust/Cargo.toml" } }
```

Comment:
`translator` module only was a part of the `hello_world` example, and it was translating words
Not sure if it's necessary in this example

Reply:
I'm pretty sure it's not :-)